webhook_handlers/views/gitlab.py

```diff
@@ -170,15 +200,15 @@ def _handle_system_hook_event(self, repo):
             message = "Repository deleted"
 
         elif event_name == "project_rename":
-            new_name = self.request.data.get("path_with_namespace").split("/")[-1]
-            repo.name = new_name
-            repo.save(update_fields=["name"])
-            message = "Repository renamed"
+            owner_username, _ = self.request.data.get("path_with_namespace").split("/")
+            self._try_initiate_sync_for_owner_by_username(owner_username=owner_username)
+            message = "Sync initiated"
 
         elif event_name == "project_transfer":
```

Comment:
we can't express this as a sync for the old owner and a sync for the new owner? i guess that wouldn't transfer historical data

Reply:
!!! I hadn't thought of that! I didn't think we had access to the old owner but i just took a look at the docs and I think I can get this field and can offload to sync! amazing

Reply:
i think changing it to do two syncs might be noticeably different - existing data from the repo on the old owner won't be available to the new owner. are we okay making that change as part of this webhook fix, or do we need to preserve how it worked before?